configs/nextflow.base.config

```diff
@@ -62,50 +62,65 @@ profiles {
 	}
 }
 
+singularity {
+	runOptions = '--bind /' // Bind directories to image for jasen run to access i.e. directories containing sample reads 
```

Comment:
Singularity already binds the current working directory, so this seems a little overkill... and perhaps having some security implications? E.g. it seems to work with only doing `--bind /tmp`